jplag/src/main/java/de/jplag/options/JPlagOptions.java

```diff
@@ -20,13 +20,17 @@
 public class JPlagOptions {
 
     private static final Logger logger = LoggerFactory.getLogger("JPlag");
-
     public static final ComparisonMode DEFAULT_COMPARISON_MODE = NORMAL;
     public static final float DEFAULT_SIMILARITY_THRESHOLD = 0;
     public static final int DEFAULT_SHOWN_COMPARISONS = 30;
 
     public static final Charset CHARSET = StandardCharsets.UTF_8;
 
+    /**
+     * The short name of the language used to parse the submissions.
+     */
```

Comment:
Adapt documentation

Reply:
mmh seems that my "search" did not find all interesting occurrences of short.

I'll fix that